### ISSUE TITLE
unused_import false positive for loadLibrary

### ISSUE THREAD

**Issue by davidmorgan** [dart-lang/linter#2905]
If an import is just used to call `loadLibrary` on a deferred library, the linter reports it as unused.

This is a false positive, it's used :)

```
import 'foo.dart' deferred as foo; // LINT (false positive)

void main() {
  foo.loadLibrary();
}
```
